brands/amenity/bank.json

```diff
@@ -2547,6 +2558,21 @@
       "name": "Clydesdale Bank"
     }
   },
+  "amenity/bank|Coast Capital Savings": {
+    "countryCodes": ["ca"],
+    "matchNames": [
+      "coast capital",
```

Comment:
is "Coast Capital" a good fit for `short_name` tag?

Reply:
I put this match in because the logo kind of deemphasizes the word “savings”, but it looks like it’s always referred to as “Coast Capital Savings”. At most it could be an `alt_name`. (In my opinion, `short_name` is best suited for acronyms and the like, as opposed to shortenings of unwieldy phrases into slightly less wieldy phrases.)

Reply:
Wiki at has

Reply:
I didn’t mean to say that `short_name` is _only_ for acronyms, but there’s a point at which a somewhat shortened name isn’t a “short name”. An extreme example would be that the `short_name` of “Doctor Martin Luther King Junior Park” should be `MLK Park`; `Doctor Martin Luther King Park;Martin Luther King Park` should go in `alt_name` rather than `short_name`. This case is not as extreme, so I don’t feel as strongly about it. In fact, I’d completely omit this name if not for the three POIs currently mistagged as `name=Coast Capital`.

By the way, the wiki mentions searching, but `short_name` is also used by some renderers in space-constrained situations.